File: frontend/src/app/QueryDto.ts
export class QueryDto {

  private _title: string;
  private _rank: number;

  constructor(title: string, rank: number) {
    this._title = title;
    this._rank = rank;
  }


  get title(): string {
    return this._title;
  }

  get rank(): number {
    return this._rank;
  }
}

File: src/main/java/hr/fer/nmbp/text_search/model/QueryDto.java
package hr.fer.nmbp.text_search.model;

public class QueryDto {

    public QueryDto(String title, double rank) {
        this.title = title;
        this.rank = rank;
    }

    private String title;
    private double rank;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public double getRank() {
        return rank;
    }

    public void setRank(double rank) {
        this.rank = rank;
    }
}
